dts/bindings/serial/nordic,nrf-uart-common.yaml

```diff
@@ -43,3 +43,29 @@ properties:
       description: |
         The CTS pin to use. The pin numbering scheme is the same as the
         tx-pin property's.
+
+    current-speed:
+      type: int
```

Comment:
type and required should be removed as they'll be set by the property definition in uart-controller.yaml